packages/application-components/src/components/internals/tabular-page.tsx

```diff
@@ -0,0 +1,65 @@
+import { css } from '@emotion/react';
+import { customProperties } from '@commercetools-uikit/design-system';
+import type { ReactNode } from 'react';
+
+type TTabControlsProps = {
+  children: ReactNode;
+};
+
+const TabControls = (props: TTabControlsProps) => (
+  <div
+    css={css`
+      margin-top: 8px;
```

Comment:
Perhaps here we could use a css variable instead.

```suggestion
      margin-top: customProperties.spacingS;
```